apps/blaze-dashboard/src/pages/setup/components/blaze-disconnected.jsx

```diff
@@ -0,0 +1,56 @@
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import privateSiteGraphic from 'calypso/assets/images/blaze/site-private-graphic@3x.png';
+
+export default function BlazeDisconnected() {
+	const translate = useTranslate();
+
+	return (
+		<>
+			<div className="promote-post-i2__inner-container">
+				<div className="promote-post-i2__setup-icon">
+					<img src={ privateSiteGraphic } alt="privete site graphic" />
```

Comment:
Same here - not sure alt text is necessary, and should be translated if it is. 

```suggestion
					<img src={ privateSiteGraphic } alt="" />
```

Reply:
thanks for the input.. you are right.. removed the alt text